packages/e2e/cypress/support/commands.ts

```diff
@@ -60,6 +60,9 @@ Cypress.Commands.add('preCompileProject', () => {
     });
     cy.request({
         url: 'api/v1/org/projects',
+        headers: {
```

Comment:
Otherwise we get a 

```
{
  "status": "error",
  "error": {
    "statusCode": 415,
    "name": "Unsupported Media Type",
    "message": "unsupported media type undefined",
    "data": [
      {
        "path": "/api/v1/org/projects",
        "message": "unsupported media type undefined"
      }
    ]
  }
}
```

response